### PR TITLE
Feature/disqus shortname env var

### PR DESCRIPTION
**Pull request by gcutrini**
Ref:
- https://tipit.avaza.com/project/view#!tab=task-panel&task=2599375
- https://tipit.avaza.com/project/view/#!tab=task-pane&task=2862172

### CODE REVIEW

**Comment by gcutrini** [comment]
This will require adding GATSBY_DISQUS_SHORTNAME var in local .env and on new Netlify deploys.

File: src/components/DisqusComponent.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ const DisqusComponent = class extends React.Component {
       <div className={className ? className : style ? '' : page === 'marketing-site' ? 'disqus-container-marketing' : 'disqus-container'} style={style}>
         {title && <span className="navbar-brand title" style={{ paddingLeft: className !== 'disqus-container-home' ? '0px' : ''}}>{title}</span>}
         <DiscussionEmbed
-          shortname='fnvirtual-poc'
+          shortname={getEnvVariable(DISQUS_SHORTNAME)}
```
Comment (smarcet) [inline src/components/DisqusComponent.js:136]: could we define a default value ?

**Review by smarcet — CHANGES_REQUESTED** [review]
@gcutrini please review comments
much thanks

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**